service/lib/agama/storage/finisher.rb

```diff
@@ -208,12 +209,40 @@ def run
 
       # Step to copy the installation logs
       class CopyLogsStep < Step
+        SCRIPTS_DIR = "/run/agama/scripts"
+
         def label
           "Copying logs"
         end
 
         def run
           wfm_write("copy_logs_finish")
+          copy_agama_scripts
+        end
+
+      private
+
+        def copy_agama_scripts
+          return unless Dir.exist?(SCRIPTS_DIR)
+
+          Yast.import "Installation"
+          require "fileutils"
+          logs_dir = File.join(Yast::Installation.destdir, "var", "log", "agama")
+          FileUtils.mkdir_p(logs_dir)
+          FileUtils.cp_r(SCRIPTS_DIR, logs_dir)
```

Comment:
We should document that the user scripts are copied to the logs in the installed system. The user scripts might contain sensitive data (passwords, tokens, keys, certificates,...) and the users should be aware of this when attaching the logs to bugzilla or sending via email.

Reply:
We could move them to a different place (in AutoYaST they are copied to `/var/adm/autoinstall`.